jmclient/jmclient/wallet.py

```diff
@@ -181,18 +208,52 @@ def add_utxo(self, txid, index, path, value, mixdepth):
 
         self._utxo[mixdepth][(txid, index)] = (path, value)
 
+    def is_disabled(self, txid, index):
+        if self._utxo_meta is None:
+            return False
+        if (txid, index) not in self._utxo_meta.keys():
```

Comment:
redundant `.keys()`

Reply:
done in 1925e0da95a1a1180431a4c3e89a35195f55d889